src/ocamlorg_frontend/pages/learn.eml

```diff
@@ -1,10 +1,51 @@
-let render ~(papers : Ood.Paper.t list) ~(release : Ood.Release.t) ~(books : Ood.Book.t list) =
+let learn_sidebar
```

Comment:
Why factor this out when it's only used once? I find the code slightly less ergonomic, and if we're not duplicating code I don't think it's very valuable either

Reply:
Ah ok, sorry I missed that! Moving it to the layout (or as a component) makes sense yes 🙂

Reply:
yes, let's make it a component, because it's not associated with the layout, but more with a subset of pages.